tests/core/_orchestrator/test_orchestrator__submit.py

```diff
@@ -562,19 +558,18 @@ def test_submit_duration_standalone_mode():
     scenario = Scenario("scenario", {task_1, task_2}, {})
     _ScenarioManager._set(scenario)
     submission = taipy.submit(scenario)
-    jobs = submission.jobs
 
-    assert_true_after_time(jobs[1].is_completed)
+    while not all(job is not None and job.is_completed() for job in submission.jobs):
+        sleep(1)  # Limit CPU usage
```

Comment:
Why not use the assert_true_after_time? It has been designed for that.

Reply:
It's because the `assert_true_after_time` wants a function.
Here, we need to check for a clause `all()` only.

Reply:
As discussed, we can try with lambda functions. 
In any case, this is not blocking.